server_admin/topics/events/login.adoc

```diff
@@ -104,5 +104,19 @@ that comes with your distribution and adding for example:
 </spi>
 ----
 
+You can also setup a maximum length of the Event detail stored in the database by editing `standalone.xml`, `standalone-ha.xml`, or 
+`domain.xml`. This can be useful in case some field (e.g. redirect_uri) is very long. There is an example how the setup can look like:
```

Comment:
Try to always follow "this" with a noun.  "This action"  Or make it "Limiting the event detail...."

Also, consider this wording: "Here is an example of defining the maximum length.: